docs/src/main/asciidoc/security-overview.adoc

```diff
@@ -17,7 +17,7 @@ Before building security into your Quarkus applications, learn about the xref:se
 == Key features of Quarkus Security
 
 The Quarkus Security framework provides built-in security authentication mechanisms for Basic, Form-based, and mutual TLS (mTLS) authentication.
-You can also use other well-known xref:security-authentication-mechanisms.adoc#other-supported-authentication-mechanisms[authentication mechanisms], such as OpenID Connect (OIDC) and WebAuthn.
+You can also use other well-known xref:security-authentication-mechanisms.adoc#other-supported-authentication-mechanisms[authentication mechanisms], such as OpenID Connect (OIDC).
```

Comment:
was it intentional to remove WebAuthn here?

Reply:
Hi Max,

Yes, it was intentional. We decided to simplify the phrase rather than create two variants using conditionalization. Users interested in the types of mechanisms available can follow the provided link for more information.

If you prefer, I can reverse this decision and include WebAuthn. What would you like to do?